components/Header.vue

```diff
@@ -13,6 +13,9 @@
       <v-btn flat to="/library" nuxt>{{ $t("menu.library") }}</v-btn>
       <v-btn flat href="https://dev.to/vuevixens" target="_blank">Blog</v-btn>
       <v-btn flat href="https://vuevixens.threadless.com/" target="_blank">{{ $t("menu.shop") }}</v-btn>
+      <v-btn flat href="https://github.com/VueVixens" target="_blank">
+        <img src="~/assets/images/github.png" alt="GitHub Logo">
```

Comment:
It's better to use the same icon we used for GitHub link in footer, i.e. `<i class="fab fa-github">`